apps/web/test/lib/checkBookingLimits.test.ts

```diff
@@ -23,20 +23,23 @@ const MOCK_DATA: Mockdata = {
 
 describe("Check Booking Limits Tests", () => {
   it("Should return no errors", async () => {
+    // @ts-expect-error Prisma v5 typings are not yet available
```

Comment:
Prisma client with extensions break Vitest deep Mocking

Reply:
Is there any estimate on when this is coming? Maybe it is worth creating a follow-up issue to tackle this when the time comes